src/physics/cam/clubb_mf.F90

```diff
@@ -1391,14 +1406,23 @@ subroutine integrate_mf( nz,
          thlflxup(k)= awthlup(k) - awup(k)*thl_env(k+1)
          qtflxup (k)= awqtup (k) - awup(k)*qt_env (k+1)
 
+         uflxup  (k)= uflxup (k) - awup(k)*u(k+1)
+         vflxup  (k)= vflxup (k) - awup(k)*v(k+1)
+

```

Comment:
The first terms on the RHS are wrong; they should be auwup(k) and avwup(k)

Reply:
These should be:

```
         uflxup  (k)= awuup(k) - awup(k)*u(k+1)
         vflxup  (k)= awvup(k) - awup(k)*v(k+1)
```

Reply:
Sorry, hopefully I have these right now.